README.md

```diff
@@ -41,3 +41,38 @@ advancement of students in science, technology, engineering, and mathematics.
 |Development hardware| No | All | Each member will require a computer capable of running the tools needed to develop the application |
 |Cloud-based hosting environment| No | Dakota (Cody) Kaup | Hosting environment to be used for development and testing|
 |CSS framework | No | Joe | A CSS framework will be used to provide additional UI controls and site theming |
+
+## Project Metholodology
+
+#### Proven Methods 
+
+WordPress 
+
+* Elegant UI out of the box, easily extendable, pluggable
+* Barrier to entry, frequent updates, complexities, not easy to test/troubleshoot, lock-in
+
+Django is 10+ year old framework that is used by companies such as Instagram and Eventbrite. Features such as:
+
+* Django provides many templates out of the box and open source CMS possibilities to meet the requirement of a minimal degree of content management. 
+
+  * Administrators must be able to easily add new content and control its metadata. 
+  * Django templates provide uniform styling for forms and pages provided to different users. For example, a form to 'gauge the user's interest' should look and feel the same way as the form to add a new event.
+
+* Built-in user management to aid in the requirement of role-based access. 
+
+* The built-in REST framework will help the application meet the requirement of being easily accessible by external sites.
```

Comment:
The Django REST framework is a library that's not included out of the box so I'd go with something like:

> Third-party libraries like the Django REST framework...